runtime/tools/python/ttrt/common/api.py

```diff
@@ -153,7 +153,8 @@ def run(args):
 
     # execution
     print("executing action for all provided flatbuffers")
-    device = ttrt.runtime.open_device(device_ids)
+    system_desc, device_ids = ttrt.runtime.get_current_system_desc()
```

Comment:
this is already called earlier on in the function, you can remove

Reply:
Great catch! This was leftover after the rebase, removed it